docs/source/docker/docker.rst

```diff
@@ -0,0 +1,100 @@
+==================================================
+NI Linux Real-Time and Docker
+==================================================
+
+.. contents:: Table of Contents
+   :depth: 2
+   :local: 
+
+Introduction
+============
+
+The `Docker platform`_ allows for developers to run applications in consistent environments across
+multiple hardware architectures and operating systems through the use of containers.
```

Comment:
I'm thinking this sentence is incorrect: Docker does not *per se* handle cross-architecture concerns, and it is usually[\*][1] rather coupled to Linux kernel features.

Perhaps we can say it helps with producing consistent environments and isolating the processes within them, and just leave it at that. Feel free to reply (or chat out-of-band) if you think more context is warranted.

[1]: <https://docs.docker.com/desktop/windows/permission-requirements/#windows-containers>